### ISSUE TITLE
typing-extensions not listed as dependency

### ISSUE THREAD

**Issue by alastairporter** [sbdchd/flake8-pie#60]
Hi!

I see that in https://github.com/sbdchd/flake8-pie/pull/56/files#diff-6dbbed115d61c90646772020134721500e433fe8f7359e4119340fa3772b07ccR7, `typing_extensions` is used, however it seems that this is a 3rd party package available on pip and not part of the python stdlib. When I install flake8-pie 0.8, flake8 fails to run with:

```
  File "/Users/xxxxxxx/env/lib/python3.9/site-packages/flake8_pie/__init__.py", line 7, in <module>
    from flake8_pie.base import Body, Error, Flake8Error
  File "/Users/xxxxxxx/env/lib/python3.9/site-packages/flake8_pie/base.py", line 6, in <module>
    from typing_extensions import Protocol
ModuleNotFoundError: No module named 'typing_extensions'

During handling of the above exception, another exception occurred:

```

**Comment by sbdchd** [comment]
Thanks for the report!

Should be fixed in 0.8.1

**Comment by alastairporter** [comment]
thanks!
